### PR TITLE
Provide android_proto_library rule

### PR DESCRIPTION
**Pull request by pcj**
This is an initial demo of android support in rules_protobuf.  It probably won't be merged directly but shows where this is going.

### CODE REVIEW

**Comment by perezd** [comment]
Is this using proto lite or the full protos?

**Comment by pcj** [comment]
Intent is to use lite, but I have not had time to research how to minimize the deps for android yet.  I think there is quite a bit of extra stuff that is ultimately not needed.

https://github.com/pubref/rules_protobuf/pull/45/files#diff-2d3e6c8e0bc02146e68201a1df759000R26

File: WORKSPACE [hunk 1]
```diff
@@ -1,5 +1,22 @@
 workspace(name = "org_pubref_rules_protobuf")
 
+
+android_sdk_repository(
+    name = "androidsdk",
+    # Replace with path to Android SDK on your system
+    path = "/Users/pcj/Library/Android/sdk",
```
Comment (kamalmarhubi) [inline WORKSPACE]: As of bazel 0.4.5 you (reliably) expect the repository rule to read the sdk location from the `ANDROID_HOME` variable.

**Comment by pcj** [comment]
Note: these rules have been re-written and migrated to https://github.com/stackb/rules_proto. Please re-open there if issue persists, thanks.

Cleaning up all issues/PRs on this repo, apologies in advance for closing without proper response.

**Pull request closed without merging**